command/daemon/config.go

```diff
@@ -61,10 +62,10 @@ type Config struct {
 	}
 
 	Resources struct {
-		LimitCPU      int64 `envconfig:"DRONE_RESOURCE_LIMIT_CPU"`
-		LimitMemory   int64 `envconfig:"DRONE_RESOURCE_LIMIT_MEMORY"`
-		RequestCPU    int64 `envconfig:"DRONE_RESOURCE_REQUEST_CPU"`
-		RequestMemory int64 `envconfig:"DRONE_RESOURCE_REQUEST_MEMORY"`
+		LimitCPU      BytesSize `envconfig:"DRONE_RESOURCE_LIMIT_CPU"`
```

Comment:
do we want this for the CPU limit and Request fields?  I thought these were whole numbers, for example `100` or `1000`

Reply:
Yeah, I was mixing it up with k8s cpu units https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-cpu

I'll update the PR.